borsh-derive/src/lib.rs

```diff
@@ -1,14 +1,24 @@
+#![recursion_limit = "128"]
+#![cfg_attr(
+    feature = "force_exhaustive_checks",
+    feature(non_exhaustive_omitted_patterns_lint)
+)]
+
 extern crate proc_macro;
 use proc_macro::TokenStream;
 use proc_macro2::Span;
 use proc_macro_crate::crate_name;
 use proc_macro_crate::FoundCrate;
 use syn::{parse_macro_input, DeriveInput, Ident, ItemEnum, ItemStruct, ItemUnion};
 
-use borsh_derive_internal::*;
-#[cfg(feature = "schema")]
-use borsh_schema_derive_internal::*;
+mod internals;
```

Comment:
add a comment, that by convention, local to `borsh-derive` crate, imports from `proc_macro` (1) are not allowed in `internal` module or in any of its submodules.